mkdocs.yml

```
@@ -0,0 +1,159 @@
+site_name: GradleUp
+site_description: >-
+  GradleUp is an umbrella GitHub organization focusing
+  on simplifying Java and Android development with Gradle.
+  On our menu: Gradle plugins, developer tools and best practices.
+copyright: >-
+  Copyright &copy; 2024 - Oleg Nenashev and all GradleUp contributors.
```

Can we just go with "GradleUp contributors"?

```suggestion
  Copyright &copy; 2024 - GradleUp contributors.
```

---

Fine by me